src/components/Inputs/CheckBox.js

```diff
@@ -10,17 +10,12 @@ export default class CheckBox extends Component {
     this.onInputChange = this.onInputChange.bind(this);
 
     }
-    onInputChange(event){
-        if(this.state.toggle){
-            this.setState({toggle:false})
-            this.props.updateCB(this.props.elementName, false);
-        }else{
-            this.setState({toggle:true})
-            this.props.updateCB(this.props.elementName, true);
-
-        }
 
+    onInputChange(_event){
+        this.setState({toggle: !this.state.toggle});
+        this.props.updateCB(this.props.elementName, !this.state.toggle);
```

Comment:
setting the state then accessing the state here might cause an issue.  Pull out the `!this.state.toggle` as a variable to avoid uncertainty.

Reply:
I resolved this potential uncertainty via https://github.com/mcode/request-generator/pull/53/commits/10f2574b83f3a58ef64e289a9db7b909a6f69800, although through just removing the internal state completely. I'm unaware of whether that will cause any performance issues, but it seems like they work fine. 

e.g. I just tried
1. changing the PIMS url (InputBox) then clicking on the "Reset PIMS" button to see the state gets passed down to and mutated by the InputBox component properly. 
2. selecting a patient and drug, clicking the "Send Rx to Pharmacy" button, then clicking the "Sign Order" button to see the rems-order-sign Request is sent in the Network tab. Then clicking the "Send Prefetch" checkbox to see whether that rems-order-sign shows up (it doesn't, good). Although unfortunately, re-checking the checkbox afterwards doesn't re-establish the connection to 127.0.0.1:8080, at least while using Docker (I'm unaware what the behavior is like just running our tech stack without Docker).